moveGoal.js

```diff
@@ -0,0 +1,33 @@
+export let yAxis = 0
+
+const randomMove = () => {
+    return Math.floor(Math.random() * (325 - -325) + -325)
```

Comment:
hmm?

325 - -325 is 325 + 325 and then +-325 is -325, so it can be shortened to:
```
Math.random() * 650 - 325
```